### DISCUSSION TITLE
i deleted my github organization by accident -- can i get back the username in less than 90 days ?

### DISCUSSION THREAD

**Discussion by ar1ja** [community/community#77043] [Discussions] [answered]
### Select Topic Area

Question

### Body

hey

i had an organization on github which i deleted by accident, now i have to wait 90 days til i can re-create it

is it possible to somehow speed up this process ?

thanks

ari

**Comment by MantasTech** [comment, marked as answer]
Hey Ari,

For your deleted GitHub org, reach out to [GitHub Support](https://support.github.com/contact) directly. They might be able to speed up the process.

Good luck!

**Comment by ar1ja** [comment]
> Hey Ari,
> 
> For your deleted GitHub org, reach out to [GitHub Support](https://support.github.com/contact) directly. They might be able to speed up the process.
> 
> Good luck!

i in fact have, even before making this post, but so far my emails and ticket has not been answered, so i thought to ask here, maybe theres some secret 1337 h4xx0r epic gamer method to speed it up somehow

ill wait a couple of days and just create a new org under a new username if i get no answer, pretty busy w school atm anyway so eh

thanks for ur answer
